build_projects/dotnet-host-build/build.ps1

```diff
@@ -8,7 +8,7 @@ param(
     [string]$Architecture="x64",
     [string]$TargetArch="",
     [string]$ToolsetDir="",
-    [string]$Framework="netcoreapp1.0",
+    [string]$Framework="netcoreapp1.1",
```

Comment:
If we are setting the TARGETFRAMEWORK enviroment variable then why do we even need the default in PrepareTargets.cs? Why not error if it isn't set?

(of course currently ignoring the fact that we are using env variables at all as I really had doing that).